src/app/docs/fluid-grid/fluid-grid-docs.component.html

```diff
@@ -7,6 +7,269 @@
+          <sky-column
+            screenSmall="3"
+          >
+            small="3"
+          </sky-column>
+          <sky-column
+            screenSmall="3"
+          >
+            small="3"
+          </sky-column>
+          <sky-column
+            screenSmall="3"
+          >
+            small="3"
+          </sky-column>
+          <sky-column
+            screenSmall="3"
+          >
+            small="3"
+          </sky-column>
+        </sky-row>
+
+        <sky-row>
+          <sky-column
+            screenSmall="4"
+          >
+            small="4"
+          </sky-column>
+          <sky-column
+            screenSmall="4"
+          >
+            small="4"
+          </sky-column>
+          <sky-column
+            screenSmall="4"
+          >
+            small="4"
+          </sky-column>
+        </sky-row>
+
+        <sky-row>
+          <sky-column
+            screenSmall="5"
+          >
+            small="5"
+          </sky-column>
+          <sky-column
+            screenSmall="7"
+          >
+            small="7"
+          </sky-column>
+        </sky-row>
+
+        <sky-row>
+          <sky-column
+            screenSmall="6"
+          >
+            small="6"
+          </sky-column>
+          <sky-column
+            screenSmall="6"
+          >
+            small="6"
+          </sky-column>
+        </sky-row>
+
+        <sky-row>
+          <sky-column
+            screenSmall="8"
+          >
+            small="8"
+          </sky-column>
+          <sky-column
+            screenSmall="4"
+          >
+            small="4"
+          </sky-column>
+        </sky-row>
+
+        <sky-row>
+          <sky-column
+            screenSmall="9"
+          >
+            small="9"
+          </sky-column>
+          <sky-column
+            screenSmall="3"
+          >
+            small="3"
+          </sky-column>
+        </sky-row>
+
+        <sky-row>
+          <sky-column
+            screenSmall="10"
+          >
+            small="10"
+          </sky-column>
+          <sky-column
+            screenSmall="2"
+          >
+            small="2"
+          </sky-column>
+        </sky-row>
+
+        <sky-row>
+          <sky-column
+            screenSmall="11"
+          >
+            small="11"
+          </sky-column>
+          <sky-column
+            screenSmall="1"
+          >
+            small="1"
+          </sky-column>
+        </sky-row>
+
+        <sky-row>
+          <sky-column
+            screenXSmall="6"
+            screenSmall="8"
+            screenMedium="9"
+            screenLarge="10"
+          >
+            screenXSmall="6" small="8" medium="9" large="10"
+          </sky-column>
+          <sky-column
+            screenXSmall="6"
+            screenSmall="4"
+            screenMedium="3"
+            screenLarge="2"
+          >
+            screenXSmall="6" small="4" medium="3" large="2"
+          </sky-column>
+        </sky-row>
+
+        <sky-row
```

Comment:
This is true to the current demo, but what's the purpose of this final row? All the other columns show sizes, but these are labelled as "First," "Second," and "Third." It's not necessarily wrong. I'm just curious if we need this row in light of what the rest of the rows are like. ... Actually, the preceding row also breaks the pattern to a lesser extent.

Also, why do the labels count down from left to write? It seems like most people's definition of the first column would be the far left column. But that label appears on the far right. I see that we have the `reverseColumnOrder` applied, but until you dig into the code, users aren't going to realize that. If we keep this row, should we turn the `reverseColumnOrder` property off?

Reply:
I'd agree with you on all points, but I'd prefer to wait for a design spec before we make any changes to avoid rework. Would you be ok with that?

Reply:
Definitely. Should I capture those questions in an issue? Or just not worry about it?